package.json

```
@@ -10,18 +10,21 @@
     "eject": "expo eject"
   },
   "dependencies": {
-    "expo": "^30.0.1",
+    "babel-plugin-transform-react-jsx-source": "^6.22.0",
+    "expo": "^33.0.0",
     "firebase": "^5.5.4",
     "react": "16.3.1",
     "react-native": "https://github.com/expo/react-native/archive/sdk-30.0.0.tar.gz",
```

```suggestion
    "react-native": "https://github.com/expo/react-native/archive/sdk-33.0.0.tar.gz",
```
My build didn't succeed until I also updated the dependency here as well